doc/wiki/move-patient-data-between-facilities.md

```diff
@@ -0,0 +1,18 @@
+# Moving patient data from one facility to another
+
+Users of Simple might need a facility's data to be transferred to another facility in some circumstances:
+- Patients were accidentally created in the wrong facility
+- A facility needs to be deleted and merged with another facility
+
+This script can assist with transferring patient data in such cases.
+```bash
+bundle exec cap india:production deploy:rake task=data_fixes:move_data_from_source_to_destination_facility[<source-facility-id>,<destination-facility-id>]
```

Comment:
Perhaps replace `india:production` with something generic? Don't want to accidentally run with that.